RadarSDK/Radar.m

```diff
@@ -838,14 +838,14 @@ + (void)autocompleteQuery:(NSString *_Nonnull)query
                    layers:(NSArray *_Nullable)layers
                     limit:(int)limit
                   country:(NSString *_Nullable)country
-              expandUnits:(BOOL)expandUnits
+                 mailable:(BOOL)mailable
```

Comment:
Let's define another `(void)autocompleteQuery` method with these like this, while also keeping the old one that takes an `expandUnits`.

For the old one, we should note in the comment that it's deprecated. We can discard the `expandUnits` (it's always true per Jeff's JS [change](https://github.com/radarlabs/radar-sdk-js/blob/fd0850e004b0952e52e2443caf550bb9676832ea/src/types.ts#L341), and pass `NO` into `mailable`.

The reason we want to do this this way is so that this isn't a breaking change that requires a migration. (People's SDKs often auto-update, for a patch/minor version bump like this here we shouldn't require code changes).

Reply:
(You'll need to have both headings in `Radar.h` as well).